platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -180,6 +179,9 @@ COPY ["hwsku.json", "/usr/share/sonic/device/x86_64-kvm_x86_64-r0/brcm_gearbox_v
 # Workaround the tcpdump issue
 RUN mv /usr/sbin/tcpdump /usr/bin/tcpdump
 
+# Backup database_config.json from package libswsscommon, restore it in start.sh
+RUN cp /var/run/redis/sonic-db/database_config.json /etc/default/sonic-db/
```

Comment:
let's not copy host version in docker-sonic-vs, it could be different

Reply:
let's modify the database_config.json

Reply:
Why could it be different?

Reply:
they are different systems, it could be different.